

fs/dcache.c: In function `__d_lookup':
fs/dcache.c:1021: warning: label `terminate' defined but not used

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/dcache.c |    1 -
 1 files changed, 1 deletion(-)

diff -puN fs/dcache.c~remove-d_bucket-warning-fix fs/dcache.c
--- 25/fs/dcache.c~remove-d_bucket-warning-fix	2004-09-26 17:20:55.730943288 -0700
+++ 25-akpm/fs/dcache.c	2004-09-26 17:20:55.734942680 -0700
@@ -1025,7 +1025,6 @@ struct dentry * __d_lookup(struct dentry
 			atomic_inc(&dentry->d_count);
 			found = dentry;
 		}
-terminate:
 		spin_unlock(&dentry->d_lock);
 		break;
 next:
_
